ci/cron/test_sha

```diff
@@ -0,0 +1 @@
+53b24793f4a90b777198c7e6dd4e509b4085c140
```

Comment:
Maybe worth giving this a more descriptive file name or move it to a `perf` subdirectory?

Reply:
how does `perf_test_sha` sound?

Reply:
Sounds great! I just want to have the `perf` part somewhere in there :slightly_smiling_face:

Reply:
moved `test_sha` and `compare-perf.sh` to `ci/cron/perf` folder.